frontend/src/views/Home.vue

```diff
@@ -2,8 +2,9 @@
   <div>
     <section class="main-content home">
       <AppLogo class="splash" :first="'Smart'" :last="'Course'"/>
-      <p>Making the most of your degree.</p>
+        <p>Community-driven ratings, reviews and info about UNSW courses</p>
```

Comment:
alignment doens't look right. I think the full-stop is better too.

Reply:
Yeah this should definitely be text-align: center. Can't really tell until u go to smaller widths.

Reply:
yeah ok